src/Apps/Legion/Hooks/useEntities.tsx

```diff
@@ -97,6 +117,8 @@ export const useEntities = () => {
          * Callback to delete the entity from state.
          * NOTE: this is not a deep update. It will only delete the root level element
          */
-        deleteEntity: deleteEntity
+        deleteEntity: deleteEntity,
+        /** Callback to get entity count per state based on Type */
+        getEntityStateCount: getEntityStateCount
```

Comment:
does this need to be a function? can't it just be an object that gets returned? The function would have to be recreated on every state update anyways so I don't think you gain much by exposing a function over the actual result itself.
Thoughts?

Reply:
I'll change the name to getEntityCounts